### ISSUE TITLE
Download buttons are not accessible when navigated to with [TAB] key

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine#1054]
The download/view links (as well as "Download all" and "Send") do not contain the name of the file that is being downloaded, viewed, or sent, making it inaccessible for a screen reader user.

Fixing this inspired some debate.

One "easy" fix would be to put the full name of each document in the button. This makes it much wider, and it also repeats the name of the document twice (for the "view" and "download" actions).

We also debated whether it was correct to use an anchor tag for this. HTML spec clearly says that links are to be used for downloads.

If we use an anchor tag, should we really style it like a button? This seems fine as long as we don't use role="button", as it's purely a visual change.

We might revisit the right fix, but for now we agreed to try a change that makes the links accessible without changing the visual style.
